### DISCUSSION TITLE
Will 2.3 on CENTOS 7 be upgradeable to 2.4?

### DISCUSSION THREAD

**Discussion by samanosuke26** [Security-Onion-Solutions/securityonion#9651] [Unsupported Versions] [answered]
I see the 2.4 is built off Rocky Linux. Will it still support CentOS 7 machines?

**Comment by InfosecGoon** [comment, marked as answer]
It will not - CentOS 7 is nearly end-of-life (June 30, 2024) and there will not be a successor to it, which is why the base install is moving to Rocky Linux.

**Reply by innovate-support** [reply]
In reading everything I can about the upcoming 2.4 release and how much is actually changing, it sounds more like it should be called 3.0

I was along for the ride when it went from 1 to 2, and this sounds very much like that.
